pkg/modules/webhook/middleware.go

```diff
@@ -173,10 +235,49 @@ func webhookMiddleware(w *Webhook) api.Middleware {
 						}
 					}
 
-					w.asyncCount.Add(1)
+					webhookSync := strings.ToLower(c.Request().Header.Get("Gotenberg-Webhook-Sync")) == "true"
```

Comment:
```suggestion
					webhookSync, err := strconv.ParseBool(c.Request().Header.Get("Gotenberg-Webhook-Sync"))
```

`ParseBool` is able to parse more "boolean" strings.

Reply:
Yep. I knew that is going to be asked. Done.